src/components/animate/AnimatePresence.vue

```diff
@@ -44,6 +44,9 @@ export default defineComponent({
     isVisible: {
       type: Boolean,
       default: () => true
+    },
+    onComplete: {
+      type: Function
```

Comment:
prob can type this one as `type: Function as PropType<(el: HTMLElement) => void>`

Reply:
yep, thanks